opthex/hex.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+import os
+import abc
+from typing import Dict
+from dataclasses import dataclass
+
+from mujoco_py import load_model_from_path, MjSim, MjViewer
+
+
+@dataclass(frozen=True)
+class HexState:
+    """Encompasses the current state of the robot."""
+
+    joints: Dict[str, float]
+    """Current joint state of the robot. A mapping between the joint name to its value."""
+
+    acceleration: np.ndarray
+    """The acceleration vector of the robot, in the robot frame."""
```

Comment:
is this 6D?

Reply:
unsure, does your accelerometer provide a 6D acceleration vector?